pocket_coffea/utils/stat/combine.py

```diff
@@ -80,9 +81,12 @@ def __init__(
         self.category = category
         self.bin_prefix = bin_prefix
         self.bin_suffix = bin_suffix
+        self.process_suffix = process_suffix
         self.verbose = verbose
         if self.bin_suffix is None:
             self.bin_suffix = "_".join(self.years)
+        if self.process_suffix is None:
+            self.process_suffix = "_".join(self.years)
```

Comment:
Isn't it better to have:
`self.process_suffix = '_' + "_".join(self.years)` ?

otherwise you might have something like`QCD2022_2023`. I think it's better to have an additional `_` between process and suffix.